test/libsolidity/syntaxTests/missing_functions_duplicate_bug.sol

```diff
@@ -1,7 +1,3 @@
-pragma solidity ^0.6.0;
-
-pragma experimental ABIEncoderV2;
-
 contract Ownable {
```

Comment:
@mijovic I think the ABIEncoderV2 part should be here. What do you think?

Reply:
The version pragma should change though.

Reply:
Usually I do not see pragmas inside our test, that is why I removed it. If you think it should be here, we should add it back :-)

Reply:
@mijovic I saw some bugs that are only triggered when ABIEncoderV2 is called; not sure if this can go.

@Marenz Do you think this change is okay?

Reply:
Don't we run all tests on both encoders?

Reply:
Ah sorry, this is a syntax test - I don't think we do those on both.

Reply:
Yes, this test needs ABIEncoderV2 because it tests some weird case where Yul functions are not added properly.